src/components/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Modal,
+  ModalFooter,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import commands from '../../commands';
+import commandsGeneral from '../../commandsGeneral';
+
+class KeyboardShortcutsModal extends React.Component {
```

Comment:
@elsenhans this looks like a good candidate for a functional component.

Reply:
I changed class component to functional component.
@mkuklis  Thanks for your feedback!